boards/arm/cxd56xx/spresense/configs/rndis/defconfig

```diff
@@ -100,7 +100,6 @@ CONFIG_NSH_DISABLE_TELNETD=y
 CONFIG_NSH_FILE_APPS=y
 CONFIG_NSH_READLINE=y
 CONFIG_NSH_TELNETD_CLIENTSTACKSIZE=3072
-CONFIG_NSH_WGET_USERAGENT="NuttX/7.2x.x (; http://www.nuttx.org/)"
```

Comment:
Is it OK to just remove this? Is there a sane default?

Reply:
See my comment in the PR.  The default is `NuttX/6.x.x (; http://www.nuttx.org/)` which is equally wrong.

Reply:
Oh ok.